extension/src/search.rs

```diff
@@ -28,14 +27,16 @@ pub fn init_table(
     // cron-like for a cron based update model, or 'realtime' for a trigger-based
     schedule: &str,
 ) -> Result<String> {
+    let table_name_str = pg_oid_to_table_name(table_name);
```

Comment:
I am hoping that we can determine the schema from the oid too. so maybe we have a `let schema = pg_oid_to_schema()` or something similar here